hy/chromebook_setup/instructions.md

```diff
@@ -0,0 +1,162 @@
+Եթե ​​Chromebook չեք օգտագործում, կարող եք անմիջապես անցնել այս բաժնի վրայով: Եթե ​​այո, ձեր տեղադրման փորձը մի փոքր այլ կլինի: Դուք կարող եք անտեսել այն տեղադրման հրահանգների մնացած հատվածը:
+
+### Cloud IDE (PaizaCloud Cloud IDE, AWS Cloud9, Glitch.com)
+
+Cloud IDE- ը գործիք է, որը Ձեզ տալիս է կոդերի խմբագրման և ինտերնետ համակարգչից օգտվելու հնարավորություն, որի վրա կարող ես տեղադրել, գրել և գործարկել ծրագրային ապահովումը: предоставляющий тебе редактор кода и доступ к компьютеру в интернете, на котором ты можешь установить, записать и выполнить программу. Այս դասընթացի շրջանակներում Cloud IDE-ին կօգտագործվի որպես Ձեր *տեղային գործիք *. Դուք աշխատելու եք տերմինալում (ինչպես և ձեր դասընկերները OS X- ում, Ubuntu- ում կամ Windows- ում), բայց ձեր տերմինալը միացված կլինելու այն համակարգչին, որը գտնվում է Նիդեռլանդներում կամ Գերմանիայում, որը cloud IDE- ը ստեղծել է հատուկ Ձեզ համար: Ահա ամպային IDE- ների ցուցումները (PaizaCloud Cloud IDE, AWS Cloud9, Glitch.com): Դուք կարող եք ընտրել ամպային IDE- ներից մեկը եւ հետեւեք IDE ամպի հրահանգին:
+
+#### PaizaCloud Cloud IDE 
+
+1. Գնալ [PaizaCloud Cloud IDE](https://paiza.cloud/)
+2. Ստեղծել account
+3. Սեղմեք *New Server* և ընտրեք Django app
+4. Սեղմեք Terminal կոճակը (պատուհանի ձախ կողմում)
+
+Այժմ դուք պետք է տեսնեք ինտերֆեյս՝ լուսանցքի նշումով և ձախ կողմում գտնվող կոճակներ: Կտտացրեք «Տերմինալ» կոճակին ՝ տերմինալի պատուհանը բացելու համար ՝ հետևյալ հաղորդագրությամբ.
+
+{% filename %}Terminal{% endfilename %}
+
+    $
+    
+
+PaizaCloud Cloud IDE- ի տերմինը պատրաստ է ձեր հրահանգներին: Կարող եք կրկնել կամ մեծացնել այդ պատուհանը, որպեսզի դա մի փոքր ավելի մեծ լինի:
+
+#### AWS Cloud9 
+
+Այժմ Cloud 9-ը պահանջում է, որ դուք գրանցվեք AWS համակարգով և մուտքագրեք կրեդիտ քարտի տվյալները:
+
+1. Ներբեռնեք Cloud 9-ը  Chrome վեբ խանութից </ 0></li> 
```

Comment:
thaere are multiple instances where there's a space between `/` and number of formatting tag. that we should resolve, otherwise the build will fail.